sweepbatcher/sweep_batch.go

```diff
@@ -749,8 +750,9 @@ func (b *batch) publishBatch(ctx context.Context) (btcutil.Amount, error) {
 	}
 
 	b.log.Infof("attempting to publish non-coop tx=%v with feerate=%v, "+
-		"totalfee=%v, sweeps=%d, destAddr=%s", batchTx.TxHash(),
-		b.rbfCache.FeeRate, fee, len(batchTx.TxIn), address)
+		"weight=%v, fee0=%v, finalfee=%v, sweeps=%d, destAddr=%s",
```

Comment:
nit: Similarly logging out `fee0` is not super informative, so perhaps we could as suggested above log something like: `calculatedfee` and `finalfee`

Reply:
Here as well: `feeForWeight` and `fee`.